### DISCUSSION TITLE
Accessing Relationship's properties

### DISCUSSION THREAD

**Discussion by itishermann** [Budibase/budibase#3673] [Help] [answered]
Hi everyone,
I'm trying to use the relationship feature with the demo Application Tracking system project where there are applicants and applicants have vacancies.
how can i get the vacancy title, for example, from the applicant object in an automation.

**Comment by mjashanks** [comment, marked as answer]
Hey @itishermann 

Assuming that you have access to the `Applicant` row, from one of your automations steps, you can use the `Query Rows` automation step to query the `Vacancy` table. i.e. `Vacancy where _id equals {{ steps.1.vacancies }} limit 1`. You can then use the output of that step in future steps.

**Reply by itishermann** [reply]
Thanks a lot

**Reply by Shyru** [reply]
I had the exact same issue and was happy to have found a solution, but I fail to find a way to enter the Query as specified above. The Query Step only allows me to specify a filter and sadly "_id" is not one of the selectable fields to filter against:
![image](https://user-images.githubusercontent.com/957556/152593862-210f7e8f-7677-4561-948c-76e2736aa96e.png)
Note I have translated my project into german. So "Stellen" matches the vacancies table, and "Stelle" is the field "vacancy" in the applicant table.
Is it perhaps that the current version implements a new filter ui and we lost the ability to specify custom queries?

**Reply by mjashanks** [reply]
@Shyru thanks for this - i've logged as a bug: https://github.com/Budibase/budibase/issues/4337

In the meantime - could you use `Auto ID` ?

**Reply by Shyru** [reply]
I tried using Auto ID, but it seems to be some other auto incremented ID, not the record ID from couchDB, so it is of no use in this situation.
